### ISSUE TITLE
data preprocessing difference between this repo and the original paper

### ISSUE THREAD

**Issue by zhijiew** [kazuto1011/deeplab-pytorch#19]
Hi! I found that you resize the images directly in data preprocessing which can change length-width ratio and it is different from the process in origin paper, and when I loaded the ckpt 'deeplabv2_resnet101_VOC2012_trainaug.pth' which converted from original caffe model and make evaluation, the miou was 74.767, it has some difference from the paper's result(76.3), I think the difference of preprocessing may cause this loss of miou. I used another [script](https://gist.github.com/zhijiew/5d39ecdb5a4d0006be385f22370c29d8) (which borrowed from this [repo](https://github.com/isht7/pytorch-deeplab-resnet)) to do the evaluation, it can get the miou of 76.24, which is very close to the result of paper

**Comment by kazuto1011** [comment]
Thank you for reporting the VOC12 results! Yes, you're right, the default DeepLab preserves the aspect ratio and processes one by one. I carefully implemented the "models" to make it can do so. However I focused on the [cocostuff-10k](https://github.com/nightrome/cocostuff10k) in this repository here, so I followed their way to reproduce the results; warping images and annotations to 513x513 in advance.
